docs/code/MlNetCookBook.md

```diff
@@ -2,6 +2,13 @@
 
 This document is intended to provide essential samples for common usage patterns of ML.NET. 
 It is advisable to be at least minimally familiar with [high-level concepts of ML.NET](MlNetHighLevelConcepts.md), otherwise the terminology in this document may be foreign to you.
+The examples in this document make use of the dynamic API, currently the supported ML.Net api.
+In ML.Net there is also a static api, that operates on the schema of the data, strongly typing the data type. That version of the API is considered experimental.
+If you are loading an existing model from a stream, there's no need to use static types (and it's also pretty hard to do). 
+Also, if the data view's schema is only known at runtime, there is no way to use static types. 
+Using the static API gives you compiler support: it's more likely that, if your code compiles, it will also work as intended.
+A version of this cookbook containing the same samples in the static API samples can be found in the [MlNetCookBook_staticApi.md](experimental\MlNetCookBook_staticApi.md)
+We welcome feedback about it.  
```

Comment:
I suggest we don't elaborate much about the static API in this doc and only limit ourselves to 1 sentence about it with the link. And push all the details (what it is, when to use it, when not to use it, advantages, motivation, etc) to the static_api.md file.

Reply:
moved that info in the static doc. 





---
In reply to: [251604799](https://github.com/dotnet/machinelearning/pull/2284#discussion_r251604799) [](ancestors = 251604799)